client/src/components/ExtraUsers/ExtraUsers.css

```diff
@@ -0,0 +1,18 @@
+.ExtraUsers {
+  background: var(--purple);
+  height: 150px;
+  width: 150px;
```

Comment:
Check with the responsive team to see if these values need to be in rem instead of px.